api/rest/v1/bulk_property_values.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: default
+title: Property Values
+nav_exclude: true
+parent: v1 REST
+grand_parent: API
+published: false
+permalink: /api/rest/v1/bulk/property/values
+---
+
+# /v1/bulk/property/values
+
+Get the values of a [property](/glossary.html#property) for multiple entities.
```

Comment:
please change "entities" to "nodes" throughout this change

Reply:
Done!